features/iframe-credentialless.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-07-11
 name: Iframe credentialless
-description: TODO
+description: When using cross-origin isolation, this allows embedding third-party iframes that do not set COEP.
```

Here's a take on the description more in the style of https://github.com/web-platform-dx/web-features/blob/main/docs/README.md#descriptions:

```suggestion
description: The `credentialless` attribute for `<iframe>` loads third-party content in an ephemeral context and does not send any credentials such as cookies. When using cross-origin isolation, this allows you to embed content that do not send COEP headers.
```

I don't know if I accurately described what it does, but I think it makes sense to say what it does in addition to the main use case for it. Also not sure if the wording around COEP is accurate, or if I broke it.

Finally, is this feature sometimes called "anonymous iframes" or is that an old and defunct name? If it is still relevant, consider adding "Also known as anonymous iframes." to the description.

---

Ah, thanks, I wasn't aware of those guidelines. Appreciate the suggestion 👍 

> Finally, is this feature sometimes called "anonymous iframes" or is that an old and defunct name?

AFAIK it's an old name. It was renamed after some initial feedback from other browser.